src/main/resources/application.yml

```diff
@@ -159,6 +171,10 @@ logging:
     org.mongodb.driver: ${LOG_LEVEL_MONGODB_DRIVER:WARN}
 
 app:
+  pdv:
+    base-url: ${PDV_DECRYPT_BASE_URL:}
+    cf: ${PDV_DECRYPT_SERVICE_CODE:pdv-ms-tokenizer}
+    headers.x-api-key: ${PDV_DECRYPT_API_KEY:x_api_key}
```

Comment:
the cf is used just for the "name" property of the FeignClient annotation, which is meaningless having it configurable
the heades.x-api-key is not used, moreover it's a constant key of this integration, thus is meaningless having it configurable
```suggestion
```

Reply:
I misunderstand the headers.x-api-key, so it was right ;)